.github/workflows/codeql-analysis.yml

```
@@ -43,7 +43,7 @@ jobs:
     - if: matrix.language == 'go'
       uses: actions/setup-go@v4
```

actionlint (1.7.11)

[error] the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)